Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -483,28 +483,15 @@ extension PledgeViewController: PledgeShippingLocationViewControllerDelegate {
 // MARK: - RewardPledgeTransitionAnimatorDelegate
 
 extension PledgeViewController: RewardPledgeTransitionAnimatorDelegate {
-  func beginTransition(_ operation: UINavigationController.Operation) {
-    self.descriptionViewController.beginTransition(operation)
-  }
-
-  func snapshotData(withContainerView view: UIView) -> RewardPledgeTransitionSnapshotData? {
-    return self.descriptionViewController.snapshotData(withContainerView: view)
-  }
+  func beginTransition(_: UINavigationController.Operation) {}
```

Comment:
Instead of deleting all of this code, I decided to keep this structure here since we discussed in standup that we may revisit the custom animation in an Investment Day.

Reply:
On second thought I think we should remove the animation, I think it's unlikely we'll make it work with this new layout.